modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-web/src/main/java/com/liferay/dynamic/data/mapping/form/web/internal/constants/DDMFormConstants.java

```diff
@@ -19,6 +19,9 @@
  */
 public class DDMFormConstants {
 
+	public static final String DDM_FORM_UPLOADED_FILES_FOLDER_NAME =
+		"Form Uploaded Files";
```

Comment:
@carolmariaabb, Luiz has updated the acceptance criteria associated to the folder's name, can you please double check this info?

Thanks.

Reply:
In fact, there was a change. Thanks @natocesarrego! 

I changed the string value to "Forms". However, I chose to keep the name of the constant as "DDM_FORM_UPLOADED_FILES_FOLDER_NAME", to be more specific and easy to change the value later, if necessary. What do you think?

Reply:
Seems to be good to me, @carolmariaabb. 🙂